demos/paired_mrus_brain/demo_train.py

```diff
@@ -47,7 +47,7 @@
 train(
     gpu="0",
     config_path=config_path,
-    gpu_allow_growth=False,
+    gpu_allow_growth=True,
```

Comment:
what is the reason for changing this?

Reply:
Make it more robust and less like to cause errors. As taking all memory might make other users/programs unable to run code.

Also, it allows you to know easily if you are using the largest batch size you can use.

Reply:
i think you mentioned this before - to be fair...